docs/mutations.md

```diff
@@ -17,7 +17,7 @@ Do you have a suggestion for a (new) mutator? Feel free to create an [issue](htt
 | `/` | `*` |
 | `%` | `*` |
 
-## Equality Operators (_equality_)
+## Equality Operators (_equality_, _patternMatching_)
```

Comment:
I also don't think we want to add patternMatching as a second name for these mutations

Reply:
Done